python/lsst/meas/base/forcedPhotCoadd.py

```diff
@@ -287,13 +287,17 @@ def runDataRef(self, dataRef, psfCache=None):
             exposure.getPsf().setCacheCapacity(psfCache)
         refCat = self.fetchReferences(dataRef, exposure)
 
-        measCat = self.measurement.generateMeasCat(exposure, refCat, refWcs,
-                                                   idFactory=self.makeIdFactory(dataRef))
+        try:
+            exposureId = coaddUtils.getExposureId(dataRef, coaddName=self.config.coaddName, includeBand=False)
+        except Exception as e:
+            self.log.warn("Setting exposureId to match Gen3 failed with: %s.  Falling back to "
```

Comment:
Please use `self.log.warning` -- what you have here will issue a deprecation warning.

Reply:
I've removed this `log.warn` and updated the one in `coadd_utils` to use `log.warning`.